src/button.js

```diff
@@ -99,6 +99,9 @@ Button.prototype.getTemplateContext = function () {
 
 Button.prototype.render = function () {
     View.prototype.render.call(this);
+    if (this.ariaLabel && !this.opts.buttonUrl) {
```

Comment:
I'm assuming that buttonUrl is an indicator for a link type of element. I searched several repos and only found it in use at https://github.com/Livefyre/streamhub-sdk/blob/724f4c688d4b6c3a2f35d8008c9742fb4092b5be/src/content/views/mixins/twitter-content-view-mixin.js#L42